docs/src/main/paradox/starvation-detector.md

```diff
@@ -0,0 +1,91 @@
+# Akka Thread Starvation Detector
+
```

Comment:
paradox is missing the file `project-info.conf`

Reply:
added